build.gradle

```diff
@@ -114,7 +114,6 @@ ext.deps = [
 
 subprojects {
     apply plugin: 'java'
-    apply plugin: 'eclipse'
```

Comment:
Can Eclipse support Azkaban code afterward?

Reply:
I am not sure. I don't use Eclipse. 
If we don't actively use it, how do we know it is working?
If there is a need for it, someone should send a PR.